modules/tls-app-v2/README.md

```diff
@@ -0,0 +1,18 @@
+# TLS app Terraform module
+
+This module defines the ACLs and quota needed by an app to communicate with an mTLS enabled kafka cluster.
+
+See [main.tf](main.tf) for what resources it defines and [variables.tf](variables.tf) for available input variables.
+
+## Usage example:
+Following the [name prefix guidelines](../../prod-aws/kafka-shared/README.md#contributing)
+ ```terraform
+ module "example" {
```

Comment:
Can we include how to set quotas in the example?

Reply:
done